lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -5,13 +5,27 @@ const _ = require('lodash');
 
 module.exports = {
   compileMethods() {
+    const corsConfig = {};
```

Comment:
Can this variable be set by the user? Because below there's a check if it contains a path.